FinishButton.vue

```diff
@@ -29,6 +35,7 @@ export default {
   },
   methods: {
     finish() {
+      this.disabled = true;
       this.$emit('finish');
```

Comment:
We're already addressing the disabled/blocked issue in PR #8. Let's get that PR fixed up, reviewed again, and merged and remove these changes here. @gannan08 -- that PR needs your attention for changes and then I can review again and merge.